doc/source/configuration/vault.rst

```diff
@@ -229,6 +229,18 @@ Enable the required TLS variables in kayobe and kolla
 
       kayobe overcloud service deploy
 
+   If VM provisioning fails with an error with this format:
```

Comment:
Did this happen after completion of the service deploy? It seems odd that it's using an HTTP endpoint.

Reply:
> Did you delete the admin endpoints before running service deploy? Should that be a separate step rather than a warning?

> Did this happen after completion of the service deploy? It seems odd that it's using an HTTP endpoint.

I did delete the admin endpoints before the service deploy, and provisioning did not work after the service deploy. I thought it was very strange that it didn't pick up the changes and needed a manual restart.

Reply:
Had the nova_compute containers been restarted during the service deploy?

Reply:
Not sure, the docker logs don't really go back far enough